cmd/agent/install_script.sh

```diff
@@ -8,6 +8,7 @@
 set -e
 install_script_version=1.1.0
```

Comment:
I suggest we release this change right away (next week) after merging, so let's bump this version.

```suggestion
install_script_version=1.2.0
```